en/animation/marionette/animation-graph-basics.md

```diff
@@ -0,0 +1,221 @@
+# Animation State Machine
+
+We call the action that an object is in that plays a particular animation, such as standby, move, run, attack, etc., **state**. <br>
+In general, an object has multiple states and switches between them in a certain logical order to perform different actions; such switches are called [transitions](state-translation.md) in the animation graph. The conditions that need to be satisfied when a transition occurs are called **transition conditions**.
+
+The animation state machine, on the other hand, is used to manage and control each state and the transition between states on the object, similar to a flowchart, which we can edit visually directly in the [Animation Graph panel](animation-graph-panel.md). Currently an animation graph contains a state machine, when the state machine is in a state in the animation graph, the animation corresponding to that state will be played. Then, according to the pre-built animation flow chart, the state machine can automatically control the playback and switching of the skeleton animation, etc.
+
+## State Basics
+
+The states in the state machine are not only used to control the animation, but also include **Entry**, **Exit**, **Any**, **Sub-State Machine**, which are special states, because they do not carry animation, but are only used to mark the start and end of the state machine. To distinguish them from other states, we call them "pseudo-states" for now.
```

Comment:
```suggestion
The states in the state machine are not only used to control the animation, but also include **Entry**, **Exit**, **Any**, **Sub-State Machine**, which are special states, because they do not carry animation, but are only used to mark the start and end of the state machine. To distinguish them from other states, they are call "pseudo-states" for now.
```

Reply:
oops, one change:
they are call "pseudo-states" for now.
-->
they are called "pseudo-states" for now.

Reply:
Fixed.